src/tests/components/ForecastSummary.test.js

```diff
@@ -1,45 +0,0 @@
-import React from "react";
-import { render } from "@testing-library/react";
-import ForecastSummary from "../../components/ForecastSummary";
-
-describe("ForecastSummary", () => {
-  const validProps = {
-    date: 1111111,
-    description: "Stub description",
-    icon: "500",
-    temperature: {
-      min: 12,
-      max: 22,
-    },
-    onSelect: () => {},
-  };
-
-  test("renders correctly", () => {
-    const { asFragment } = render(
-      <ForecastSummary
-        date={validProps.date}
```

Comment:
Here you can either destructure all of these from validProps or have them defined as their own variables. e.g.
```
const description="asdfghj"
const date = 44
```